src/cider/nrepl/middleware/inspect.clj

```diff
@@ -77,7 +77,9 @@
     (inspector-response msg (swap-inspector! msg #(inspect/refresh % overrides)))))
 
 (defn- toggle-view-mode [{:keys [view-mode] :as inspector}]
-  (let [toggle-order {:normal :object, :object :normal}
+  (let [toggle-order (if (inspect/supports-table-view-mode? inspector)
```

Comment:
I'd add a note here that depends on the underlying data, as I'm not sure it will be obvious from the name of the function alone.